Dockerfile

```diff
@@ -21,17 +21,22 @@ RUN apk --update --no-cache add \
     shadow \
     su-exec \
     tzdata \
+    postgresql-dev \
   && apk --update --no-cache add -t build-dependencies \
     build-base \
     git \
     libffi-dev \
     libressl-dev \
+    gcc \
+    python-dev \
+    musl-dev \
   && git clone https://github.com/mozilla-services/syncserver app \
   && cd app \
   && git reset --hard $SHA1_COMMIT \
   && pip install pymysql \
   && pip install --upgrade --no-cache-dir -r requirements.txt \
   && pip install --upgrade --no-cache-dir -r dev-requirements.txt \
+  && pip install psycopg2 \
```

Comment:
Instead of adding a new line you can replace `&& pip install pymysql \`
with `&& pip install pymysql psycopg2 \` ?